.editorconfig

```diff
@@ -0,0 +1,14 @@
+# editorconfig.org
+
+root = true
+
+[*]
+end_of_line = lf
+indent_style = tab
```

Comment:
Want to earn 20% more income from the comfort of your own home?

https://stackoverflow.blog/2017/06/15/developers-use-spaces-make-money-use-tabs/

Reply:
:grin: I did that because our Makefile is using tabs (which I think is a make requirement). So I put tab here and add all other files that use spaces as an exception a few lines below.

Reply:
Yeah Makefile but afaik it's the only language that enforces this requirement, and if we are to specify a default I vote spaces with Makefile as the exception.

Reply:
fixed :)

Reply:
🙇